mkdocs.yml

```
@@ -123,6 +123,7 @@ nav:
       - 'Observability': authorino/docs/user-guides/observability.md
     - 'Rate Limiting':
       - 'RateLimitPolicy for Application Developers': multicluster-gateway-controller/docs/how-to/simple-ratelimitpolicy-for-app-developers.md
+      - 'Authenticated Rate Limiting for Application Developers': kuadrant-operator/doc/user-guides/authenticated-rl-for-app-developers.md
       - 'RateLimitPolicy for Platform Engineers': kuadrant-operator/doc/user-guides/gateway-rl-for-cluster-operators.md
```

I hate myself. Sorry @Boomatang. Do you mind grouping these 2 together?

There's a continuity between 1st and 2nd on this list, then the 2 others will be about authenticated RL (first a simpler one, then a more complex one.)

```suggestion
      - 'RateLimitPolicy for Platform Engineers': kuadrant-operator/doc/user-guides/gateway-rl-for-cluster-operators.md
      - 'Authenticated Rate Limiting for Application Developers': kuadrant-operator/doc/user-guides/authenticated-rl-for-app-developers.md
```